tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/testdatatypes/CsoAccountGuid.java

```diff
@@ -0,0 +1,32 @@
+package ca.bc.gov.open.jagefilingapi.qa.testdatatypes;
+
+public class CsoAccountGuid {
+
+    private String validExistingCSOGuid;
+    private String invalidGuid;
```

Comment:
I still think it is better to name this as:
```
invalidNoFilingRoleGuid
```
or something like that so it is clear

Reply:
Updated. Thanks!